### PR TITLE
Samjd

### PR DESCRIPTION
**Pull request by sditkovsky**
migrated the `woa13()` and helper functions in `datasets.py.` Set up a new folder within notebooks to test out the aguadv workflow. Successfully performed binning operation on woa data.

### CODE REVIEW

**Comment by sditkovsky** [comment]
The issue did start when I added cmip6_preprocessing to the CI env (via pip). Any chance that's what's making your other tests act up as well?

**Comment by jbusecke** [comment]
I doubt it since all the tests are hung up on the [setup-miniconda action](https://github.com/jbusecke/cmip6_omz/pull/1/checks?check_run_id=2776147670)... 

This is really frustrating...

**Comment by sditkovsky** [comment]
I think pip was the issue. Tests run when I use conda-forge to install `cmip6_preprocessing`. Now it's just not finding `cmip6_dataset_id` in `cmip6_preprocessing.utils`. Is this an issue with the version of the package on conda-forge? @jbusecke

**Comment by jbusecke** [comment]
Yeah I think that feature is not released, I am having some trouble with the pip publishing. Let me prioritize that now.

**Comment by sditkovsky** [comment]
Should I switch back to pip for installing cmip6_preprocessing?

**Comment by jbusecke** [comment]
I **just** merged the new version '0.3.2' to conda-forge. I have restarted the jobs to see if that fixes it. 

Sorry for the delay on that, getting this release to work was quite the nightmare, haha.

**Comment by sditkovsky** [comment]
No worries! Thanks for bearing with me on this first pull request. I know it got a little bloated. I'll aim for trimmer PRs going forward

File: cmip6_omz/datasets.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+from cmip6_preprocessing.preprocessing import correct_lon
+
+from omz_tools import convert_o2_ml_l
+
+# TODO: We are planning on integrating this functionality in xgcm at a later point. Check back in time to see if we can refactor this.
+from xarrayutils.utils import dll_dist
+
+
+def reconstruct_areacello(ds):
```
Comment (jbusecke) [inline cmip6_omz/datasets.py:16]: I think this should probably live in another module? It should probably also get a test. We can do that in another PR, but should we make an issue so we don't forget?

File: notebooks/sam_tests/.ipynb_checkpoints/test_conda_env-checkpoint.ipynb [hunk 2]
```diff
@@ -0,0 +1,306 @@
+{
```
[the remark this answers is not in the document]
Comment (jbusecke) [reply notebooks/sam_tests/.ipynb_checkpoints/test_conda_env-checkpoint.ipynb]: I am going ahead and remove it from the PR, ok?

**Review by jbusecke — APPROVED** [review]
Great job @sditkovsky. Sorry that this dragged out for so long, due to my cmip6_pp backlog.  
Ill go ahead and merge.

File: cmip6_omz/datasets.py (hunk 1 above)
Comment (sditkovsky) [reply cmip6_omz/datasets.py:16]: Sounds good to me. I'll post the issue
